src/database/filter/__tests__/Filters.test.ts

```diff
@@ -0,0 +1,113 @@
+import { Filters } from '../Filters';
+describe("Filters", () => {
+
+    describe("sender Token", () => {
+        test("add new senderToken", () => {
+            const filters = new Filters();
+    
+            filters.addSenderToken("eth", 10);
+    
+            expect(filters.senderToken).toEqual({ "eth": { min: 10, max: 10 } });
+        });
```

Comment:
```suggestion
describe("Filters", () => {
    let filters: Filters;

    beforeEach(() => {
        filters = new Filters();
    });

    describe("sender Token", () => {
        test("add new senderToken", () => {
            filters.addSenderToken("eth", 10);
    
            expect(filters.senderToken).toEqual({ "eth": { min: 10, max: 10 } });
        });
```

Reply:
This way you can remove all the `const filters = new Filters();` in your tests.

Reply:
Can do, but it breaks the pattern Arrange/Act/Assert

Reply:
done